lib/include/platform/iot_metrics.h

```diff
@@ -87,10 +87,19 @@
         #define IotMetrics_FreeTcpConnection    free
     #endif
 
+    #ifndef IotMetrics_MallocIpAddress
+        #define IotMetrics_MallocIpAddress    malloc
```

Comment:
all malloc/free functions should be defined to pvPortMalloc and pvPortFree, the FreeRTOS wrappers for malloc and free
whatever else is wrong, even in the libraries...
this is a difference between C-SDK code and a:FR code...

Reply:
For example, check this file
c:\src2\amazon-freertos\tests\common\include\iot_config_common.h
for the test config... we should have the same setup as this.

Reply:
Thanks for catching this. I missed it. Let me update iot_config_common.h